src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -236,6 +236,7 @@ public async Task<bool> TryRefreshAsync(CancellationToken cancellationToken)
             return true;
         }
 
+        [Obsolete("SetDirty will be deprecated in a future release. Please use 'ProcessPushNotification' method instead.")]

```

Comment:
Is this necessary given the class is internal?

Reply:
You're right. This is not required in internal classes.